src/query/mod.rs

```diff
@@ -48,6 +55,43 @@ impl fmt::Display for Timestamp {
     }
 }
 
+#[cfg(any(test, feature = "chrono_timestamps"))]
+impl Into<DateTime<Utc>> for Timestamp {
+    fn into(self) -> DateTime<Utc> {
+        match self {
+            Timestamp::Now => Utc::now(),
+            Timestamp::Hours(h) => {
+                let millis = h * 60 * 60 * 1000;
+                Utc.timestamp_millis(millis.try_into().unwrap())
+            }
+            Timestamp::Minutes(m) => {
+                let millis = m * 60 * 1000;
+                Utc.timestamp_millis(millis.try_into().unwrap())
+            }
+            Timestamp::Seconds(m) => {
+                let millis = m * 1000;
+                Utc.timestamp_millis(millis.try_into().unwrap())
+            }
+            Timestamp::Milliseconds(millis) => Utc.timestamp_millis(millis.try_into().unwrap()),
+            Timestamp::Nanoseconds(ns) => Utc.timestamp_nanos(ns.try_into().unwrap()),
+            Timestamp::Microseconds(mis) => {
+                let nanos = mis / 10000;
+                Utc.timestamp_nanos(nanos.try_into().unwrap())
+            }
+        }
+    }
+}
+
+#[cfg(any(test, feature = "chrono_timestamps"))]
+impl<T> From<DateTime<T>> for Timestamp
+where
+    T: TimeZone,
+{
+    fn from(date_time: DateTime<T>) -> Self {
+        Timestamp::Milliseconds(date_time.timestamp_millis() as usize)
```

Comment:
Better use `Timestamp:Nanoseconds` here, according to my comment.

Reply:
Also, [`timestamp_millis`](https://docs.rs/chrono/0.4.9/chrono/offset/trait.TimeZone.html#method.timestamp_millis) can panic while `timestamp_nanos` can't.